### PR TITLE
fixed local storage issue with different meal types

### PR DESCRIPTION
**Pull request by TaylorAkin**
@samanthamcq   If you could review this that would be great, thanks!

### CODE REVIEW

File: src/MenuItems.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ class MenuItems extends React.Component {
```
Comment (samanthamcq) [inline src/MenuItems.js:31]: If you can remove some extra blank lines, it will look better.

File: src/Restaurant.js [hunk 2]
```diff
@@ -23,11 +23,9 @@ const styles = {
   var gitDomain = 'taylorakin.github.io';
 
   var currentDomain = document.domain;
-  console.log(currentDomain);
-
 
   if(currentDomain === gitDomain){
-    console.log('ta');
+  
     path = "/react_restaurant";
```
Comment (samanthamcq) [inline src/Restaurant.js:29]: You can use double quote or single quote for string.  It's better to be consistent. So choose one and use the same style for the whole project.

var gitDomain = 'taylorakin.github.io';
path = "/react_restaurant";

File: src/Restaurant.js [hunk 3]
```diff
@@ -62,13 +60,13 @@ class ParallaxMenu extends React.Component {
...
       <div style={{ height: 500 }}>
-        <ConditionalButton name= 'Breakfast'/>
+        <ConditionalButton name= 'Breakfast' numOfItems='6' />
       </div>
     </Parallax>
     <h1 id = 'lunch'>| | |</h1>
     <Parallax bgImage={image3} strength={-100}>
       <div style={{ height: 500 }}>
-      <ConditionalButton  name= 'Lunch'/>
+      <ConditionalButton  name= 'Lunch' numOfItems='8'/>
```
Comment (samanthamcq) [inline src/Restaurant.js:69]: It's better to add indentation to the component under the div.

**Review by samanthamcq — COMMENTED** [review]
It's very good.  Only minor comments about the style and formatting.

File: src/MenuItems.js (hunk 1 above)
Comment (TaylorAkin) [reply src/MenuItems.js:31]: Will do!

File: src/Restaurant.js (hunk 2 above)
Comment (TaylorAkin) [reply src/Restaurant.js:29]: Thanks

**Pull request merged**